js/ccf-app/src/global.ts

```diff
@@ -102,16 +103,30 @@ export interface Receipt {
  */
 export interface HistoricalState {
   /**
-   * The ID of the transaction.
+   * The ID of the transaction, formatted as '<view>.<seqno>' string.
    */
   transactionId: string;
 
   /**
    * The receipt for the historic transaction.
    */
   receipt: Receipt;
+
+  /**
+   * An object that provides access to the maps of the Key-Value Store
+   * associated with the historic transaction.
+   * Fields are map names and values are {@linkcode KvMap} objects.
+   */
+  kv: KvMaps;
```

Comment:
Currently, this is unused for `"mode": "historical"` endpoints since the historical kv is already in `ccf.kv`. To satisfy the API it could be populated as well, though it would be redundant. But as at least the type doesn't have to be changed to be `| undefined`.